src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -1133,6 +1133,12 @@ def to_snowpark(
             index, index_label
         )
 
+    def cache_result(self) -> "SnowflakeQueryCompiler":
+        """
+        Materializes and caches this query compiler in place.
```

Comment:
I am confused about the inplace here, how is this inplace?

Reply:
Good catch - old comment I forgot to delete, thank you!